src/magic_folder/test/test_snapshot.py

```diff
@@ -45,8 +47,13 @@
     create_local_author_from_config,
     write_local_author,
     create_snapshot,
+    LocalSnapshot,
 )
+from allmydata.util.fileutil import abspath_expanduser_unicode
 
```

Comment:
IMO we should avoid any new `allmydata.*` imports; can we use `FilePath` instead to get whatever functionality we wanted from this?